cli/lsp/config.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::AnyError;
+use deno_core::serde::Deserialize;
+use deno_core::serde_json;
+use deno_core::serde_json::Value;
+
+#[derive(Debug, Clone, Default)]
+pub struct ClientCapabilities {
+  pub status_notification: bool,
+}
+
+#[derive(Debug, Clone, Default, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Settings {
```

Comment:
Nitpick: `UserSettings`?

Reply:
That is confusing in my opinion... as in vscode "user" settings are the global settings...  maybe `WorkspaceSettings` is more clear, which is what I have changed it to.